pkg/genericapiserver/options/server_run_options.go

```diff
@@ -73,17 +77,18 @@ func NewServerRunOptions() *ServerRunOptions {
 			Prefix: DefaultEtcdPathPrefix,
 			DeserializationCacheSize: DefaultDeserializationCacheSize,
 		},
-		EnableLogsSupport:    true,
-		EnableProfiling:      true,
-		EnableWatchCache:     true,
-		InsecureBindAddress:  net.ParseIP("127.0.0.1"),
-		InsecurePort:         8080,
-		LongRunningRequestRE: defaultLongRunningRequestRE,
-		MasterCount:          1,
-		MaxRequestsInFlight:  400,
-		MinRequestTimeout:    1800,
-		RuntimeConfig:        make(config.ConfigurationMap),
-		SecurePort:           6443,
+		DefaultStorageMediaType: "application/json",
```

Comment:
That would require this package to take a dependency on runtime, which
pulls in a larger portion of the stack.  I would prefer not to do that here.

On Mon, May 9, 2016 at 2:18 AM, Nikhil Jindal <notifications@github.com>
wrote:

> In pkg/genericapiserver/options/server_run_options.go
> <https://github.com/kubernetes/kubernetes/pull/25266#discussion_r62453651>
> :
>
> > @@ -73,17 +77,18 @@ func NewServerRunOptions() *ServerRunOptions {
> >  			Prefix: DefaultEtcdPathPrefix,
> >  			DeserializationCacheSize: DefaultDeserializationCacheSize,
> >  		},
> > -		EnableLogsSupport:    true,
> > -		EnableProfiling:      true,
> > -		EnableWatchCache:     true,
> > -		InsecureBindAddress:  net.ParseIP("127.0.0.1"),
> > -		InsecurePort:         8080,
> > -		LongRunningRequestRE: defaultLongRunningRequestRE,
> > -		MasterCount:          1,
> > -		MaxRequestsInFlight:  400,
> > -		MinRequestTimeout:    1800,
> > -		RuntimeConfig:        make(config.ConfigurationMap),
> > -		SecurePort:           6443,
> > +		DefaultStorageMediaType: "application/json",
>
> Use runtime.ContentTypeJSON
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/25266/files/b115989fa2a41611e476c986f1a7e742b15af291#r62453651>
>